src/plugins/github/README.md

```diff
@@ -0,0 +1,212 @@
+
+### Repo
+
+A GitHub repository, e.g. [sourcecred/sourcecred]. It will be directly
+connected to all of the pulls and issues in the repository. The repo node has
+no timestamp, so setting a weight on the repository will have no effect (i.e.
+they do not mint cred). This will change when we switch to [CredRank].
+
+[CredRank]: https://github.com/sourcecred/sourcecred/issues/1686
+
+[sourcecred/sourcecred]: https://github.com/sourcecred/sourcecred
+
+### Issue
+
+A GitHub issue, e.g. [sourcecred/sourcecred#40]. Issues are connected to their
+author(s), to entities that they reference, to their comments, and to the
+containing GitHub repository.
+
+[sourcecred/sourcecred#40]: https://github.com/sourcecred/sourcecred/issues/40
+
+### Pull Request
+
+A GitHub pull request, e.g. [sourcecred/sourcecred#35][pull]. Pulls are
+connected to their author(s), to entities they reference, to their comments,
+their reviews, to their containing repository, and, (if merged) to the commit
+that they merged as.
+
+[pull]: https://github.com/sourcecred/sourcecred/pull/35
+
+### Pull Request Review
+
+A review of a GitHub pull request, e.g. [this review]. Reviews are connected to
+their author(s), to entities they reference, to their comments, and to the pull
+they review. Note that review assignments are not currently tracked.
+
+
+[this review]: https://github.com/sourcecred/sourcecred/pull/91#pullrequestreview-105254836
+
+### Comment
+
+A comment on an issue, pull request, or pull request review. Comments are
+connected to their author(s), to entities they reference, and to their 'parent'
+(the containing issue, pull, or review).
+
+### Commit
+
+A commit is a Git commit, as discovered via the GitHub API, e.g. [this commit].
+We currently enumerate every commit that is in the history of the `master`
+branch.
+
+Commits currently do not add much value in the cred graph, because they are not
+meaningfully connected to the contents of the commits. For example, it would be
+great if a code module flowed cred to commits that implemented or modified the
+module, which could then flow to the pull requests that added those commits. In
+this case, we would be making meaningful use of having commits in the graph.
+
+However, as of March 2020, we do not track cred at the module, file, or
+directory level. Because of this, commits do not add much value beyond their
+connections to pull requests (which are already in the graph).
+
+[this commit]: https://github.com/sourcecred/sourcecred/commit/94b04541514b991c304616aadfcb417a19871e82
+
+### User
+
+A GitHub user account, e.g. [@decentralion]. User accounts do not mint cred, so
+setting a node weight would have no effect. Using the identity plugin, it's
+possible to "collapse" user nodes with other identity nodes into a single,
+canonical identity. For example, if a contributor had a GitHub user account and
+a Discourse account, then the identity plugin can collapse those identities
```

Comment:
Do we want to say that identities are merged in a JSON file, or is that more "technical documentation" than conceptual overview?

Reply:
Let's save that for the identities plugin.